src/filePush/filePush.ts

```diff
@@ -11,40 +12,42 @@ export interface Config {
   privateKey?: string;
 }
 
-export const createConfig = async () => {
-  const config: Config = {
-    host: process.env.SFTP_Address,
-    username: process.env.SFTP_User,
-    retries: 3,
-  };
+type Feeds = 'evl' | 'tfl';
+
+export const createConfig = async (eventType: string) => {
+  let secretString: string;
 
-  if (process.env.SFTP_Key && process.env.SFTP_Key != '') {
-    const sftpKey = await getSecret(process.env.SFTP_Key);
-    config.privateKey = sftpKey;
-  } else if (process.env.SFTP_Password && process.env.SFTP_Password != '') {
-    const sftpPassword = await getSecret(process.env.SFTP_Password);
-    config.password = sftpPassword;
+  if (eventType === 'evl') {
+    secretString = await getSecret(process.env.EVL_SFTP_CONFIG);
+  } else if (eventType === 'tfl') {
+    secretString = await getSecret(process.env.TFL_SFTP_CONFIG);
   } else {
-    logger.error(
-      '',
-      'No password or private key found, please check the env variables',
-    );
-    throw new Error(
-      'No password or private key found, please check the env variables',
-    );
+    logger.error('', 'Unable to determine event type, please try again');
   }
 
-  return config;
+  try {
+    return JSON.parse(secretString) as Config;
+  } catch (e) {
+    logger.error('', e);
```

Comment:
Can probably resolve that issue you were seeing in the logs by adding 

```suggestion
    logger.error("Error parsing secret, see callstack below")
    logger.error('', e);
```